src/lib/components/ComponentIndex/Card.svelte

```diff
@@ -95,22 +95,22 @@
       <Tag title={tag} variant='blue' />
     {/each}
   </div>
-  <div class="card__bottom">
-    <div>
-      {#if stars > 0}
+  {#if stars > 0}
```

Comment:
0 is different than `undefined`. can we still show it if `0` but no `undefined`? maybe change to `{#if typeof stars !== 'undefined'}

Reply:
Well spotted ! 
Tested it locally, seems fine : 
![image](https://user-images.githubusercontent.com/23191482/131407395-0a9de98b-495a-4149-8866-9b635c14d7b7.png)